test/Faker/Provider/ng_NG/PersonTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Faker\Test\Provider\ng_NG;
+
+use Faker\Generator;
+use Faker\Provider\ng_NG\Person;
+use Faker\Test\Provider\ng_NG\MainTest;
+
+class PersonTest extends MainTest
+{
+     /**
+     * @var Generator
+     */
+    private $faker;
+
+    public function setUp()
+    {
+        $faker = new Generator();
+        $faker->addProvider(new Person($faker));
+        $this->faker = $faker;
+    }
+
+    public function testFirstName()
```

Comment:
`testFirstName` what? The test title should mention what you test, contain a verb